doc/source/rllib/rllib-algorithms.rst

```diff
@@ -932,8 +931,8 @@ as the module uses the values from the paper by default. For example:
         # exploration, e.g. EpsilonGreedy for DQN, StochasticSampling for PG/SAC).
         "sub_exploration": {
```

Comment:
I think this overlaps with my efforts in https://github.com/ray-project/ray/pull/29796. Don't want to create merge conflicts here!